cmd/server.go

```diff
@@ -74,7 +75,8 @@ func init() {
 	serverCmd.Flags().DurationVar(&ownerFlushInterval, "owner-flush-interval", time.Millisecond*200, "owner flushes changefeed status interval")
 	serverCmd.Flags().DurationVar(&processorFlushInterval, "processor-flush-interval", time.Millisecond*100, "processor flushes task status interval")
 
-	serverCmd.Flags().IntVar(&numConcurrentWorker, "sorter-num-concurrent-worker", 8, "sorter concurrency level")
+	serverCmd.Flags().IntVar(&numWorkerPoolGoroutine, "sorter-num-workerpool-goroutine", 16, "sorter workerpool size")
+	serverCmd.Flags().IntVar(&numConcurrentWorker, "sorter-num-concurrent-worker", 4, "sorter concurrency level")
```

Comment:
Each heap sorter registers into a worker in the `heapSorterPool`, if the default worker pool size is larger than heap sorter concurrent number, will some goroutines in the worker pool do nothing?

Reply:
The sorter concurrency number is per table. So if the total number of heap sorter instances in the process is greater than or equal to the pool size, all worker goroutines in the pool are guaranteed to do something.

Reply:
Please add some comments about what is the proper value of these two variables

Reply:
Except for some rare instances, all tables whose handles are assigned to a given worker goroutine have been deleted or otherwise terminated. We do not support rebalancing for now.